ttt-game/src/components/board.js

```diff
@@ -0,0 +1,102 @@
+import React, {useReducer, useEffect, useContext} from 'react'
+import PlayerOne from './player-one.js'
+import PlayerTwo from './player-two.js'
+import TimeMachine from './TimeMachine.js'
+import { ScoreContext } from '../contexts/ScoreContext.js'
+
+const board = [null,null,null,null,null,null,null,null,null]
+let turn = 0
+let timeSteps = []
+
+
+  
+const Board = () => {
+  const [score, setScore] =  useContext(ScoreContext)
```

Comment:
👍 I really like how you separate the `score` from the other board state via the context. Keeps your concerns nice and should scale well.